curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo/64fefebad99209211ec30537.md

```diff
@@ -0,0 +1,400 @@
+---
+id: 64fefebad99209211ec30537
+title: Step 48
+challengeType: 0
+dashedName: step-48
+---
+
+# --description--
+
+If you check the `Application` tab of your browser console, you'd see that what you have there is a series of `[object Object]`. That's because everything you save in `localStorage` has to be a string.
```

Comment:
```suggestion
If you check the `Application` tab of your browser console, you'll see a series of `[object Object]`. That's because everything you save in `localStorage` has to be a string.
```